internal/stanza/converter.go

```diff
@@ -55,6 +55,24 @@ func Convert(obsLog *entry.Entry) pdata.Logs {
 
 	insertToAttributeVal(obsLog.Body, lr.Body())
 
+	if obsLog.TraceId != nil {
+		var buffer [16]byte
+		copy(buffer[0:16], obsLog.TraceId)
```

Comment:
Do we copy because `obsLog.TraceId` may not be 16 bytes? Should we change obsLog.TraceId type to be [16]byte and enforce the size? This should be both more type safe and faster. Similarly for SpanId.

Can be done in a separate PR, what you have is fine for now.

Reply:
I will do this in a separate PR, as it requires changes in the `log-collector`.